### PR TITLE
[Documentation] Updated CHANGELOG

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose

Recent PRs did not update changelog, so as asked by @JoshuaLai I am updating this document to reflect recent changes.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
```
N/A
```

* Test the code

```
N/A
```

## What to Check
Verify that the following are valid
```
N/A
```
## Other Information

```
N/A
```

### CODE REVIEW

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+## TBA (upcoming release)
+### Bugs Fixed
+- Fixed issue where demo app can't dismiss setting page when in landscape mode. [#280](https://github.com/Azure/communication-ui-library-ios/pull/280)
+- Fixing speaking overly custom colour [#281](https://github.com/Azure/communication-ui-library-ios/pull/281)
+- Fixed the one UITest failure reported in app center about setting text in acs text field [#282](https://github.com/Azure/communication-ui-library-ios/pull/282)
+- Fixed issue in uitest that app is not able to fully dismiss the settings page [#293](https://github.com/Azure/communication-ui-library-ios/pull/293)
+- Fixed Participant Drawer Height Calculation [#297](https://github.com/Azure/communication-ui-library-ios/pull/297)
+### Other Changes
+- Update PR template [#283](https://github.com/Azure/communication-ui-library-ios/pull/283)
+- Unit tests retain cycle fix [#284](https://github.com/Azure/communication-ui-library-ios/pull/284)
```
Comment (jpeng-ms) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:12]: #284 sounds like a bug fix, but it was marked as refactor, so I put it under "other changes"

Comment (vhuseinova-msft) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:14]: I don't think that we should include Unit test fix here at all. We usually don't include all PRs in our Changelog but 
- the bugs/changes that are included in Known issues or are important to end users
- features
- breaking changes
- changes in public api.
Please see `Change log` page for more details.
@jpeng-ms @JoshuaLai were there discussions to change this behaiour?

Comment (JoshuaLai) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:14]: Nope. That makes sense I think we can remove the PR template change, the unit test fix, convert redux actions, and the read me updates from the `CHANGELOG`.

Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md]: Thanks @JoshuaLai @vhuseinova-msft , Updated!

**Review by vhuseinova-msft — APPROVED** [review]

**Review by JoshuaLai — APPROVED** [review]

**Pull request merged**